[DOCS] Adjusts note on minimum node size for ELSER and third-party models (#2447)
elastic/stack-docs: docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

To use ELSER, you must have the {subscriptions}[appropriate subscription] level 
for semantic search or the trial period activated.

NOTE: The minimum dedicated ML node size for deploying and using the ELSER model 
is 4 GB in Elasticsearch Service with 

> {cloud}/ec-autoscaling.html[deployment autoscaling] turned on. Autoscaling 

Reviewer: ```suggestion
{cloud}/ec-autoscaling.html[deployment autoscaling] turned off. Autoscaling is recommended because it
```

Reviewer 2: I got a bit confused here. Don't we recommend 4GB **and** Autoscaling turned **on**?

Reviewer 2: Ok, now I got it. You mean "IF you have autoscaling turned OFF, then you should use min 4GB. If you have autoscaling ON, then it takes care of node sizing itself, so don't worry"

allows your deployment to automatically adjust its available resources when 

> required. Better performance can be achieved by using bigger ML nodes.

Reviewer: ```suggestion
required. Better performance can be achieved by using more allocations or more threads per allocation, which will require bigger ML nodes. Autoscaling will provide bigger nodes when required. If not using autoscaling you must provide suitably sized nodes yourself.
```

To use ELSER, you must have the {subscriptions}[appropriate subscription] level 
for semantic search or the trial period activated.

NOTE: The minimum dedicated ML node size for deploying and using the ELSER model 
is 4 GB in Elasticsearch Service if 
{cloud}/ec-autoscaling.html[deployment autoscaling] is turned off. Turning on 

> autoscaling is recommended because it allows your deployment to dinamically 

Reviewer: ```suggestion
autoscaling is recommended because it allows your deployment to dynamically 
```

adjust resources based on demand. Better performance can be achieved by using 
more allocations or more threads per allocation, which requires bigger ML nodes. 

> Autoscaling provide bigger nodes when required. If autoscaling is turned off, 

Reviewer: ```suggestion
Autoscaling provides bigger nodes when required. If autoscaling is turned off, 
```
